CONTRIBUTING.md

```diff
@@ -361,15 +362,15 @@ There are a few other build options available:
 ### Deprecation policy
 
 Qiskit Experiments is part of Qiskit and, therefore, the [Qiskit Deprecation
-Policy](https://qiskit.org/documentation/deprecation_policy.html) fully applies here.
+Policy](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md) fully applies here.
```

Comment:
I was concerned that this policy might not apply to experiments now with 1.0 but `DEPRECATION.md` just contains the same content that we were linking to (which is hard to find now that the old link redirects to `DEPRECATION.md` but you can see it [here](https://qiskit-org-web-resources.s3.us.cloud-object-storage.appdomain.cloud/documentation/stable/0.44/deprecation_policy.html)). I think `DEPRECATION.md` might still get an update for 1.0.

Should we decouple from the Qiskit repo by copying more of the old policy over here? We don't need to do it in this PR.

Reply:
Good point, I'll open a new PR for adding our own `DEPRECATION.md` based on the current Qiskit one.

Reply:
Perhaps we could still link to the Qiskit one rather than copying all of it and say something like "Changes and removals of public facing interfaces must be preceded by a deprecation warning in at least one previous minor release published at least three months before the change. The Qiskit deprecation guide has good advice on how to deprecate a feature. Follow it when possible, but note that Qiskit Experiments has a different release model than Qiskit." I think we want to take out the "part of Qiskit and the Qiskit policy fully applies here" part.

Reply:
Done in #1385.